apps/nsqd/options.go

```diff
@@ -131,6 +131,8 @@ func nsqdFlagSet(opts *nsqd.Options) *flag.FlagSet {
 	flagSet.String("https-address", opts.HTTPSAddress, "<addr>:<port> to listen on for HTTPS clients")
 	flagSet.String("http-address", opts.HTTPAddress, "<addr>:<port> to listen on for HTTP clients")
 	flagSet.String("tcp-address", opts.TCPAddress, "<addr>:<port> to listen on for TCP clients")
+	flagSet.Bool("use-unix-sockets", opts.UseUnixSockets, "Usinx UNIX sockets instead of IP sockets")
```

Comment:
Instead of adding a new flag, can you detect whether the `tcp-address` or `http-address` flags value is a unix socket path instead of an ip:port (and allow them to be tcp or unix socket individually)?

(should also update the help text to say that they can be "<addr>:<port> or unix /path" or similar)

Reply:
Done